packages/compass-sidebar/src/components/connections-filter-popover.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useCallback, type PropsWithChildren } from 'react';
+
+import {
+  css,
+  Icon,
+  IconButton,
+  InteractivePopover,
+  Label,
+  Overline,
+  palette,
+  spacing,
+  Toggle,
+  Tooltip,
+  useId,
+} from '@mongodb-js/compass-components';
+import type { ConnectionsFilter } from './use-filtered-connections';
+
+const containerStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'flex-start',
+  padding: spacing[300],
+  minWidth: 270,
+});
+
+const closeButtonStyles = css({
+  // An alternative to this is to pass hideCloseButton to InteractivePopover,
+  // but that throws an error when the popover is opened
```

Comment:
I don't see any errors logged or thrown when running this locally and passing a prop, can you clarify what errors do you see?

Reply:
I have no idea why I was seeing that error. It's gone for me now as well. Perhaps I had a stale build of `@mongodb-js/compass-components` or something 🤔 I've pushed a commit passing `hideCloseButton` instead 👍